dev/Repeater/SelectionModel.cpp

```diff
@@ -638,7 +638,8 @@ void SelectionModel::SelectWithPathImpl(const winrt::IndexPath& index, bool sele
     // Selection is actually different from previous one, so update.
     if (newSelection)
     {
-        bool changedSelection = false;
+        // If we unselect something, raise event any way, otherwise changedSelection is false
```

Comment:
Why? If we end up calling Select(x, false) and x was not selected then we shouldn't raise a selection changed event right?

Reply:
Oh, yes missed the whole deselect scenario. Added tests for that now and fixed any falsely raised selection changed events for deselecting already deselected nodes.